0-bootstrap/main.tf

```diff
@@ -43,15 +43,18 @@ resource "google_folder" "seed" {
 }
 
 module "seed_bootstrap" {
-  source                  = "terraform-google-modules/bootstrap/google"
-  version                 = "~> 1.0"
+  source = "github.com/terraform-google-modules/terraform-google-bootstrap"
+  # version                 = "~> 1.0"
```

Comment:
is this supposed to be commented out?

Reply:
@rjerrems this was temporary as I was waiting for the new 1.2 bootstrap release, will revert.

Reply:
It looks like this is still commented out?

Reply:
just fixed :)